OJP/OJP_FareSupport.xsd

```diff
@@ -257,7 +257,7 @@
 		<xs:sequence>
 			<xs:element name="FareProductId" type="FareProductCodeType">
 				<xs:annotation>
-					<xs:documentation>Unique Id of the FareProduct.</xs:documentation>
+					<xs:documentation>Unique Code of the FareProduct.</xs:documentation>
```

Comment:
This should only be part of https://github.com/VDVde/OJP/pull/267 I guess

Reply:
forgot to remove it